scripts/tests/ingest-fp.test.sh

```
@@ -0,0 +1,268 @@
+
+has "規定: フォーマット契約が fp の入力を外部 Issue 本文と定めている" "$FORMAT_DOC" \
+    "取得したての外部 Issue 本文"
+
+# --- 移行完了条件・比較不能の扱い・PR2 の実施条件（CodeRabbit #133 指摘 1） ---
+# 移行の窓は一度きりで、取りこぼしは「静かに」起きる（更新検知の不在は観測できない）。
+# fail-closed であること——判定できないものを「移行済み」に読み替えないこと——を両ファイルで固定する。
+has "移行: 台帳側から駆動すると明記している（取得不能を訪問漏れにしない）" "$SKILL_MD" \
+    "移行は台帳側から駆動する"
+has "移行: 比較不能の分類がある" "$SKILL_MD" "比較不能"
+has "移行: 比較不能では人間記入欄も fp も更新しないと明記している" "$SKILL_MD" \
+    "`fp` を書き換えず、人間記入欄も更新しない"
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)